src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -32,18 +32,15 @@ namespace System.Data.Common
     [System.Security.Permissions.SecurityPermission(System.Security.Permissions.SecurityAction.InheritanceDemand, ControlEvidence = true, ControlPolicy = true)]
     public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
     {
-        [System.Obsolete("DBDataPermission() has been deprecated.  Use the DBDataPermission(PermissionState.None) constructor.  http://go.microsoft.com/fwlink/?linkid=14202", true)]
```

Comment:
@danmosemsft didn't mean to remove the Obsolete attribute completely but to remove the text from it so that only "[Obsolete]" is left.